TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java

```diff
@@ -93,18 +93,23 @@ public void onClick(View v) {
         insert.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                id = id + uid1ET.getText().toString();
+                id = uid1ET.getText().toString();
```

Comment:
Please revert the change here as by adding `id` to we will know the type whether it is `Donation` or `Registration`  then check if the length equals to 3

Reply:
@mustaqmustu I did not notice that change. Please revert that in another pr ASAP.

Reply:
I reverted the PR please make another PR